src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileFields.tsx

```diff
@@ -44,164 +53,196 @@ interface SettingsEditProfileFieldsProps {
+                data-testid="edit-profile-full-name-input"
+              />
+
+              <LocationAutocompleteInput
+                title="Primary Location"
+                placeholder="City name"
+                name="location"
+                maxLength={256}
+                spellCheck={false}
+                defaultValue={values.displayLocation.display ?? undefined}
+                onSelect={(place?: Place) => {
+                  setFieldValue("location", normalizePlace(place, false))
+                }}
+                onChange={() => {
+                  setFieldValue("location", {})
+                }}
+              />
+
+              <Input
+                title="Profession"
+                placeholder="Profession or job title"
+                name="profession"
+                maxLength={256}
+                onChange={handleChange}
+                value={values.profession}
+                data-testid="edit-profile-profession-input"
+              />
+
+              <Input
+                title="Other relevant positions"
+                placeholder="Other relevant positions"
+                name="otherRelevantPositions"
+                maxLength={256}
+                onChange={handleChange}
+                value={values.otherRelevantPositions}
+                data-testid="edit-profile-other-relevant-positions-input"
+              />
+
+              <Input
+                title="About"
+                placeholder="Add a brief bio, so galleries know which artists or genres you collect."
+                name="bio"
+                multiple
+                maxLength={150}
+                onChange={handleChange}
+                value={values.bio}
+                data-testid="edit-profile-about-input"
+              />
+            </Join>
+
+            <Spacer y={[4, 6]} />
+
+            <Box>
+              <Flex alignItems="center">
+                <CheckCircleIcon fill="black60" mr={0.5} />
+                <Text variant="sm-display">Verify Your ID</Text>
+              </Flex>
+
+              <Text variant="sm" mt={1} color="black60">
+                For details, see FAQs or contact verification@artsy.net
+              </Text>
+            </Box>
+
+            <Spacer y={[4, 6]} />
+
+            <Box>
+              <Flex alignItems="center">
+                <CheckCircleFillIcon fill="green100" mr={0.5} />
+                <Text variant="sm-display">Email Address Verified</Text>
+              </Flex>
+
+              <Text variant="sm" mt={1} color="black60">
+                For details, see FAQs or contact verification@artsy.net
+              </Text>
+            </Box>
+
+            <Button
+              mt={6}
+              px={4}
+              width={["100%", "auto"]}
+              data-testid="edit-profile-save-button"
+              type="submit"
+              size="large"
+              variant="primaryBlack"
+              loading={isSubmitting}
```

Comment:
**suggestion:** Not sure if you need to add this line or if Formik disables the button automatically 🤔 

```suggestion
              loading={isSubmitting}
              disabled={!isValid}
```